lizrd/scripts/grid.py

```diff
@@ -43,16 +41,16 @@ def create_subprocess_args(
     git_branch,
     neptune_key,
     wandb_key,
-    CLUSTER_NAME,
+    CLUSTER,
     skip_confirmation=False,
     skip_copy_code=False,
 ):
-    configs = prepare_configs(config_path, git_branch, CLUSTER_NAME)
+    configs = prepare_configs(config_path, git_branch, CLUSTER)
     grid = setup_experiments(configs)
     check_for_argparse_correctness(grid)
     interactive_debug_session = grid[0][0]["interactive_debug_session"]
 
-    if CLUSTER_NAME != MachineBackend.LOCAL and not skip_confirmation:
+    if not isinstance(CLUSTER, LocalBackend) and not skip_confirmation:
```

Comment:
nie jestem fanem tych patternów, powinienem wywalić je do metody Backendu? Jeśli tak, to czy trzymać w Backendzie  `args.skip_confirmation`?